.github/workflows/performance-metrics.yml

```diff
@@ -43,4 +43,5 @@ jobs:
                   CWD: '${{ steps.generate_app.outputs.project_path }}'
                   PROJECT: q4-pwa-perf-develop
                   TARGET: production
-                  FLAGS: --wait --cloud-origin https://cloud-testing.mobify-staging.com
+                  CLOUD_ORIGIN: ${{ vars.MRT_STG_CLOUD_ORIGIN }}
```

Comment:
Now we're no longer hardcoding the cloud-origin. Furthermore, I was using the wrong value before.. but now it's correct.